File: porfolio_angular_spring_boot/backend_porfolio/src/main/java/com/example/demo/Entity/Proyecto.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.example.demo.Entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

/**
 *
 * @author Aylen
 */
@Getter @Setter
@Entity
public class Proyecto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombreP;
    private String descripcionP;
    private String urlP;

    public Proyecto() {
    }

    public Proyecto(String nombreP, String descripcionP, String urlP) {
        this.nombreP = nombreP;
        this.descripcionP = descripcionP;
        this.urlP = urlP;
    }
    
     
}

File: porfolio_angular_spring_boot/frontend_porfolio/src/app/models/proyecto.ts
export class Proyecto {
    id: number;
    nombreP: String;
    descripcionP: String;
    urlP: String;

    constructor(nombreP: String, descripcionP: String, urlP: String){
        this.nombreP = nombreP;
        this.descripcionP = descripcionP;
        this.urlP = urlP;
    }
}
